app/views/goodsProfile/GoodsRecordsView.scala.html

```diff
@@ -56,40 +60,126 @@
+                            <input class="govuk-checkboxes__input" id="status-ready" name="filterRecordStatus"
+                                   type="checkbox" value="immi-ready">
+                            <label class="govuk-label govuk-checkboxes__label" for="status-ready">
+                                @messages("declarableStatus.immiReady")
+                            </label>
+                        </div>
+                        <div class="govuk-checkboxes__item">
+                            <input class="govuk-checkboxes__input" id="status-not-ready" name="filterRecordStatus"
+                                   type="checkbox" value="not-immi-ready">
+                            <label class="govuk-label govuk-checkboxes__label" for="status-not-ready">
+                                @messages("declarableStatus.notReadyForImmi")
+                            </label>
+                        </div>
+                        <div class="govuk-checkboxes__item">
+                            <input class="govuk-checkboxes__input" id="status-action-needed" name="filterRecordStatus"
+                                   type="checkbox" value="action-needed">
+                            <label class="govuk-label govuk-checkboxes__label" for="status-action-needed">
+                                @messages("declarableStatus.notReadyForUse")
+                            </label>
+                        </div>
+                    </div>
+
+                </fieldset>
 
-                @govukButton(
-                    ButtonViewModel(messages("site.search")).withAttribute("style", "margin-left: 10px")
+                @govukSelect(
+                SelectViewModel(
+                field = form("value"),
+                items = selectItems(countries),
+                label = LabelViewModel(messages("goodsRecords.tableHeader.countryOfOrigin")).withSize(Small)
                 )
+                .withCssClass("autocomplete")
+                .withAttribute(("lang", "en"))
+                )
+
+                <div class="govuk-button-group">
+
+                    @govukButton(
+                    ButtonViewModel(
+                    content = HtmlContent(
+                    s"""${messages("site.apply")}<span class="govuk-visually-hidden">${messages("goodsRecords.record.search")}</span>"""
+                    )
+                    ).withAttribute("style", "margin-left: 10px")
+                    )
+
+                    <a class="govuk-link" href="#">@messages("site.clear")<span class="govuk-visually-hidden"> @messages("goodsRecords.record.clear")</span></a>
+                </div>
             </div>
-        }
+
+            <div class="govuk-grid-column-two-thirds govuk-!-padding-left-20">
+                <h2 class="govuk-heading-m">@messages("goodsRecords.h2.1")</h2>
+
+                <p class="govuk-body">@messages("goodsRecords.p1")</p>
+
+                <p class="govuk-body">@messages("goodsRecords.p2")
```

Comment:
Potential refactor could be to use the paragraphBody component we have then pass in a paragraph and a govukTag component as a argument which could potentially make the messages keys just be @paragraphBody(messages(goodsRecords.p2.1, govukTag)